model/database/common.yaml

```
@@ -21,3 +21,24 @@ groups:
           The `error.type` SHOULD match the error code returned by the database or the client library,
           the canonical name of exception that occurred, or another low-cardinality error identifier.
           Instrumentations SHOULD document the list of errors they report.
+
+  - id: attributes.db.cosmos.minimal
```

we use `db.cosmosdb` everywhere, lets change it to `cosmosdb` here too

```suggestion
  - id: attributes.db.cosmosdb.minimal
```